src/Symfony/Component/Security/Core/Authorization/Voter/AccessTrait.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authorization\Voter;
+
+/**
+ * @author Dany Maillard <danymaillard93b@gmail.com>
+ */
+trait AccessTrait
```

Comment:
What is the goal of that trait ? And should it really be part of the public API of the component ?

Reply:
huhm, I'm not sure (since this is really not my code. I only merge it to 5.4. I see it is being used in 2 other files: AccessDecision and Vote. Should I simply copy the code over to those 2 files? Should be quick.

Reply:
To me, yes, you should copy it. The code is simple getter, so there is no harm duplicating it. And it avoids having to maintain BC on that trait.